shard.yml

```diff
@@ -9,8 +9,11 @@ crystal: 0.25.0
 license: MIT
 
 dependencies:
+  # selenium:
+  #   github: paulcsmith/selenium-webdriver-crystal
+  #   tag: crystal-v0.25
   selenium:
-    github: paulcsmith/selenium-webdriver-crystal
-    tag: crystal-v0.25
+    github: DmitryBochkarev/selenium-webdriver-crystal
+    branch: windows-implementation-1
```

Comment:
Looks like it was merged in. Could you point this to the official repo: https://github.com/ysbaddaden/selenium-webdriver-crystal now that it has crystal 0.25 support and your fullscreen changes?